snapshots/un/2023-10-09/plastic_waste_1999_2010.csv.dvc

```diff
@@ -0,0 +1,40 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: 'UN Comtrade: Plastic Waste Trade (UN, 2023)'
```

Comment:
Please take a look at the definitions for the different fields in `origin`: https://docs.owid.io/projects/etl/architecture/metadata/reference/origin/.

In your case, I suggest using also `title_snapshot` and `description_snapshot`, where you can refer to the specific subset of data that you've downloaded.

In the `title`, we should not include the source name or year of publication, this is information already included in fields like `producer`, `attribution` and/or `attribution_short`, and `date_published`.

In your case, I'd use

```yaml
title: Comtrade Database
description: |-
	...
title_snapshot: Comtrade Database: Plastic Waste Trade, 1999-2010
description_snapshot: |-
	Subset of Comtrade database only concerning indicators related to ...

	This snapshot only includes data from 1999 until 2010.
```